administrator/components/com_admin/sql/updates/postgresql/5.1.0-2024-03-28.sql

```diff
@@ -0,0 +1,6 @@
+--
+-- Add post-installation message about Brotli compression in .htaccess
+--
+INSERT IGNORE INTO "#__postinstall_messages" ("extension_id", "title_key", "description_key", "action_key", "language_extension", "language_client_id", "type", "action_file", "action", "condition_file", "condition_method", "version_introduced", "enabled")
```

Comment:
```suggestion
INSERT INTO "#__postinstall_messages" ("extension_id", "title_key", "description_key", "action_key", "language_extension", "language_client_id", "type", "action_file", "action", "condition_file", "condition_method", "version_introduced", "enabled")
```
`INSERT IGNORE` is MySQL/MariaDB syntax and will cause a syntax error on PostgreSQL.

The same thing for PostgreSQL is already done by the `ON CONFLICT DO NOTHING` at the end of the query.

Reply:
my bad copy paste